.eslintrc.js

```diff
@@ -58,8 +58,7 @@ module.exports = {
     "no-self-compare": "error",
     "no-throw-literal": "error",
     "no-unmodified-loop-condition": "error",
-    // "no-unused-expression": "error", // to be enabled after we upgrade eslint
-    // "no-use-before-define": "error", // to be enabled after fixing issues in our code
```

Comment:
I removed no-use-before-define because it's triggered quite a lot in our code, and I don't find it brings a lot of value.